### ISSUE TITLE
Keep getting: You're not supposed to be here!

### ISSUE THREAD

**Issue by nasheayahu** [ntop/ntopng#6589]
pfSense
2.6.0-RELEASE (amd64)
built on Mon Jan 31 19:57:53 UTC 2022
FreeBSD 12.3-STABLE

Was wonder why I could not connect, had to go to the direct console to repeating messages, after rebooting was directed to view the Crash Reporter and got this:

```
Crash report begins.  Anonymous machine information:

amd64
12.3-STABLE
FreeBSD 12.3-STABLE RELENG_2_6_0-n226742-1285d6d205f pfSense

Crash report details:

No PHP errors found.

Filename: /var/crash/info.0
Dump header from device: /dev/ada0p3
  Architecture: amd64
  Architecture Version: 4
  Dump Length: 74752
  Blocksize: 512
  Compression: none
  Dumptime: Wed May 11 16:58:38 2022
  Hostname: pfSense.kohanyim.com
  Magic: FreeBSD Text Dump
  Version String: FreeBSD 12.3-STABLE RELENG_2_6_0-n226742-1285d6d205f pfSense
  Panic String: 
  Dump Parity: 2187160421
  Bounds: 0
  Dump Status: good
  ...
  ..
  .
```

I posted the on Netgate "[The firewall has encountered an error](https://forum.netgate.com/topic/172117/the-firewall-has-encountered-an-error/3?_=1652354214201) " and got a response of, "The message buffer shows Snort crashing out".  I already sent Snort a email regarding this, but since this incident I've been getting this message "You're not supposed to be here!" when I want to view Flow and this is what is displayed:

```
Internal server error

__.local/share/ntopng/scripts/lua/modules/plugins_utils.lua:1056: module 'active_monitoring.am_utils' not found:__no field package.preload['active_monitoring.am_utils']__no file '/var/db/ntopng/plugins0/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/var/db/ntopng/plugins0/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/notifications/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/alert_store/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/toasts/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/recipients/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/var/db/ntopng/plugins0/alert_definitions/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/alert_definitions/other/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/alert_definitions/host/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/pro/scripts/lua/enterprise/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/pro/scripts/lua/enterprise/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/alert_definitions/flow/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/alert_keys/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/notifications/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/notifications/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/var/db/ntopng/plugins0/ts_schemas/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/timeseries/custom/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/timeseries/schemas/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/timeseries/drivers/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/var/db/ntopng/plugins0/http_lint/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/notifications/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/flow_dbms/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/timeseries/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/i18n/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/pools/active_monitoring/am_utils.lua'__no file '/usr/local/share/ntopng/scripts/lua/modules/active_monitoring/am_utils.lua'__no file '/usr/local/share/lua/5.3/active_monitoring/am_utils.lua'__no file '/usr/local/share/lua/5.3/active_monitoring/am_utils/init.lua'__no file '/usr/local/lib/lua/5.3/active_monitoring/am_utils.lua'__no file '/usr/local/lib/lua/5.3/active_monitoring/am_utils/init.lua'__no file './active_monitoring/am_utils.lua'__no file './active_monitoring/am_utils/init.lua'__no file '/usr/local/lib/lua/5.3/active_monitoring/am_utils.so'__no file '/usr/local/lib/lua/5.3/loadall.so'__no file './active_monitoring/am_utils.so'__no file '/usr/local/lib/lua/5.3/active_monitoring.so'__no file '/usr/local/lib/lua/5.3/loadall.so'__no file './active_monitoring.so'
```

How do I fix this, do I have to re-install?

**Comment by MatteoBiscosi** [comment]
Hi @nasheayahu do you get that error when you try to jump to this page?
![Screenshot from 2022-05-12 10-04-31](https://user-images.githubusercontent.com/49585191/168022622-5c65ce3e-4d53-46f9-9b86-a27e8b9d82b1.png)

**Comment by nasheayahu** [comment]
It gives me the message when I select the IP Address on Hosts:

- All Hosts, 
- Networks | Networks Name, 
- Host Pool List | Pool Name | All Hosts for Pool Default, 
- Autonomous Systems | AS Number | All Hosts for AS 396982, 
- Countries | All Hosts for Country US, 
- Hosts by Operating System | Name | All Linux Hosts, 
- VLANs | All Hosts [VLAN 1077], 
- Local HTTP Servers | HTTP Server IP, 
- Top Hosts (Local).

**Comment by MatteoBiscosi** [comment]
Hi @nasheayahu Could you please update to the latest version and let me know if it is still bugged?

**Comment by nasheayahu** [comment]
Sure but, its currently showing 0.8.13_10 when will new update take effect on pfSense?

**Comment by MatteoBiscosi** [comment]
Hi @nasheayahu you are not using our ntopng, you are using ntopng version shipped by pfsense (0.8.13_10).
Please follow the documentation and install our repository https://www.ntop.org/guides/ntopng/third_party_integrations/pfsense.html?highlight=pfsense
Our ntopng current version is the 5.3 one.
In that version there should not be this issue (tested it right now); Thanks for your report
